apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/MutateRowsTest.cs

```diff
@@ -282,5 +282,33 @@ public class MutateRowsTest
                         }
                     }));
         }
+
+        [Fact]
+        public async Task OmittedVersion()
+        {
+            var beforeInsertTimestamp = DateTime.UtcNow.AddMilliseconds(-1);
+
+            var tableName = _fixture.TableName;
+            var client = _fixture.TableClient;
+            BigtableByteString rowKey = Guid.NewGuid().ToString();
+            var response = await client.MutateRowsAsync(
+                tableName,
+                Mutations.CreateEntry(
+                    rowKey,
+                    Mutations.SetCell(
+                        BigtableFixture.DefaultColumnFamily,
+                        "column_name",
+                        "test12345")));
+
+            var entries = response.Entries.OrderBy(e => e.Index);
+            Assert.True(entries.All(e => e.Status.Code == (int)Code.Ok));
```

Comment:
I think there may be an `Assert.All` available - I'm not sure. (Worth checking.)

Reply:
I think I looked into this at one point, but decided against it. Assert.All takes an action delegate, not a func used as a predicate, so you'd have to do something like this:
```csharp
Assert.All(entries, e => Assert.Equal((int)Code.Ok, e.Status.Code));
```
Assert.True(...) seemed just slightly less ugly. But if you feel strongly about it I can change it.

Reply:
I don't feel strongly about it - but if it fails, I think Assert.All would give more information about the actual value. No biggie either way though.

Reply:
Oh that's a good point. I've used this in a few places. I'll fix them all up in a followup PR.